src/main/resources/com/dynatrace/jenkins/dashboard/Messages.properties

```diff
@@ -17,7 +17,8 @@ RECORDER_VALIDATION_BLANK_PASSWORD=Password for REST interface cannot be empty
 RECORDER_VALIDATION_BLANK_SYSTEM_PROFILE=System profile name cannot be empty
 RECORDER_VALIDATION_CONNECTION_OK=Connection successful
 RECORDER_VALIDATION_CONNECTION_UNAUTHORIZED=Connection failure - user is not authorized
-RECORDER_VALIDATION_CONNECTION_FORBIDDEN=Connection failure - Test Center Edition license is not installed
+RECORDER_VALIDATION_CONNECTION_FORBIDDEN_HTTPS_USED=Connection forbidden - probably Test Center Edition license is not installed
+RECORDER_VALIDATION_CONNECTION_FORBIDDEN_HTTP_USED=Connection forbidden - probably HTTP connections are not permitted on Dynatrace server or Test Center Edition license is not installed
```

Comment:
Connection forbidden - validate your AppMon Server settings (if HTTP connections are permitted), license (Test Center Edition license is required), or check error details